copier/cli.py

```diff
@@ -80,35 +80,32 @@ def _handle_exceptions(method: Callable[[], None]) -> int:
     return 0
 
 
-class CopierApp(cli.Application):
+class CopierApp(cli.Application):  # type: ignore[misc]
     """The Copier CLI application."""
 
     DESCRIPTION = "Create a new project from a template."
-    DESCRIPTION_MORE = (
-        dedent(
-            """\
+    DESCRIPTION_MORE = dedent(
+        """\
```

Comment:
These formatting changes again :smile: 
I suppose we should review and merge #1557 first, to get rid of this once and for all.

Reply:
ha i'm really struggling with this dev environment. keeps picking up my system version of ruff and applying these changes!

Reply:
or i can clean them up manually when everything else is reviewed, i don't really mind

Reply:
I think you can also disable auto-formatting in your editor settings while working on this (be careful not to include changes to VSCode's settings.json in the PR though). Unless you mean that the precommit tasks use your system Ruff instead of the Copier environment one? In any case, don't bother too much, this is not fun and I can revert it myself before we merge :slightly_smiling_face: